sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -236,7 +284,6 @@ static rom_error_t mask_rom_boot(const manifest_t *manifest,
   sec_mmio_check_counters(/*expected_check_count=*/4);
```

Comment:
Given the new set of register writes, it is probably a good idea to consider moving the `sec_mmio_check_values(rnd_uint32())` check right before this call.

Reply:
done